internal/target/query.go

```diff
@@ -3,46 +3,51 @@ package target
 const (
 	setChangesQuery = `
 with
-set_libraries (library_id) as (
+set_sources (source_id, type) as (
   -- returns the SET list
-  select public_id
-    from credential_library
+  select public_id, type
+    from credential_source_all_types
    where public_id in (%s)
 ),
-current_libraries (library_id) as (
+current_sources (source_id, type) as (
   -- returns the current list
-  select credential_library_id
+  select credential_library_id, 'library'
     from target_credential_library
    where target_id          = @target_id
      and credential_purpose = @purpose
+  union
+  select credential_static_id, 'static'
```

Comment:
We have `credential_library_id`, shouldn't we have a similar `credential_id`? For instance it's not `select credential_library_id, 'vault'`, it's `library`. So it feels like this should be `select credential_id, 'credential'`.

Reply:
I think it depends on where this is being selected from. `credential_library_id` is from `target_credential_library`, while `credential_static_id` is from `target_static_credential`, either way they both end up populating `source_id` for the `current_sources` cte.